connectivity/FEATURE_BLE/source/cordio/source/PalPrivateAddressControllerImpl.cpp

```diff
@@ -19,24 +19,32 @@
 
 #include "PalPrivateAddressControllerImpl.h"
 #include "dm_api.h"
+#include "mbed-trace/mbed_trace.h"
+#include "common/ble_trace_helpers.h"
+
+#define TRACE_GROUP "BLSM"
```

Comment:
maybe it's worth using a different tag: BLPR?

Reply:
Agreed.